eng/azure-pipelines.yml

```diff
@@ -34,10 +34,10 @@ jobs:
       pool: 
         ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
           name: NetCorePublic-Pool
-          queue: buildpool.windows.10.amd64.vs2017.open
```

Comment:
lolwut

Reply:
well none of our tests depended on the install environment so I just picked the queue with the largest pool so CI would be as fast as possible. Now that they do I will need to binary search to find one that has F# installed so we can load legacy projects